--- a/src/qtel/MainWindow.cpp
+++ b/src/qtel/MainWindow.cpp
@@ -166,11 +166,11 @@ MainWindow::MainWindow(void)
   station_view->addAction(addSelectedToBookmarksAction);
   station_view->addAction(removeSelectedFromBookmarksAction);
 
-  connect(directoryRefreshAction, SIGNAL(activated()),
+  connect(directoryRefreshAction, SIGNAL(triggered()),
       	  this, SLOT(refreshCallList()));
-  connect(connectionConnectToSelectedAction, SIGNAL(activated()),
+  connect(connectionConnectToSelectedAction, SIGNAL(triggered()),
       	  this, SLOT(connectionConnectToSelectedActionActivated()));
-  connect(connectionConnectToIpAction, SIGNAL(activated()),
+  connect(connectionConnectToIpAction, SIGNAL(triggered()),
       	  this, SLOT(connectionConnectToIpActionActivated()));
   connect(addSelectedToBookmarksAction, SIGNAL(triggered()),
 	  this, SLOT(addSelectedToBookmarks()));
@@ -178,16 +178,16 @@ MainWindow::MainWindow(void)
 	  this, SLOT(removeSelectedFromBookmarks()));
   connect(addNamedStationToBookmarksAction, SIGNAL(triggered()),
 	  this, SLOT(addNamedStationToBookmarks()));
-  connect(settingsConfigureAction, SIGNAL(activated()),
+  connect(settingsConfigureAction, SIGNAL(triggered()),
       	  this, SLOT(settings()));
-  connect(helpAboutAction, SIGNAL(activated()),
+  connect(helpAboutAction, SIGNAL(triggered()),
       	  this, SLOT(helpAbout()));
 
   connect(station_view_selector, SIGNAL(currentItemChanged(QListWidgetItem*,
 						           QListWidgetItem*)),
 	  this, SLOT(stationViewSelectorCurrentItemChanged(QListWidgetItem*,
 						           QListWidgetItem*)));
-  connect(station_view, SIGNAL(activated(const QModelIndex&)),
+  connect(station_view, SIGNAL(triggered(const QModelIndex&)),
 	  this, SLOT(stationViewDoubleClicked(const QModelIndex&)));
   connect(incoming_con_view, SIGNAL(itemSelectionChanged()),
       	  this, SLOT(incomingSelectionChanged()));
--- a/src/qtel/MainWindowBase.ui
+++ b/src/qtel/MainWindowBase.ui
@@ -465,7 +465,7 @@
  <connections>
   <connection>
    <sender>fileQuitAction</sender>
-   <signal>activated()</signal>
+   <signal>triggered()</signal>
    <receiver>MainWindowBase</receiver>
    <slot>close()</slot>
    <hints>
